Cargo.toml

```diff
@@ -7,7 +7,14 @@ members = [
 
 [workspace.package]
 version = "0.21.1"
+authors = ["Kampfkarren <kampfkarren@gmail.com>"]
+categories = ["command-line-utilities"]
+keywords = ["lua", "linter", "lua51"]

```

Comment:
Don't include categories and keywords, actually. selene-lib isn't really meant to be advertised.